lakey_service/downloader/executors/baseexecutor.py

```diff
@@ -0,0 +1,56 @@
+
+class BaseExecutor:
+
+    def execute(self, download_request):
+        raise NotImplementedError()
+
+    def get_sample(self, catalogue_item):
+        raise NotImplementedError()
+
+    def get_size(self, column_name, catalogue_item):
+        raise NotImplementedError()
+
+    def get_distribution(
+            self, column_name, column_type, column_is_enum, catalogue_item):
+        raise NotImplementedError()
+
+    def compile_to_query(self, download_request):
+
+        table = download_request.catalogue_item.name
```

Comment:
[NOTE FOR FUTURE] for sure we must add one more field to store table / collection name for a given catalog item name and the one we have right now use as a display name